addons/web/static/src/core/notifications/notification.xml

```diff
@@ -12,7 +12,7 @@
                 <div t-if="props.message" class="me-auto o_notification_content" t-out="props.message"/>
                 <div t-if="props.buttons.length" class="mt-2 o_notification_buttons">
                     <button t-foreach="props.buttons" t-as="button" type="button" t-key="button_index"
-                    t-attf-class="btn ps-0 {{button.primary ? 'btn-primary' : 'btn-link'}}" t-on-click="button.onClick">
+                    t-attf-class="btn {{button.primary ? 'btn-primary' : 'btn-secondary'}} me-2" t-on-click="button.onClick">
```

Comment:
@anso-odoo 
LGTM, but I wonder if it wouldn't be better to use `d-flex gap-2` on `o_notification_buttons` instead of margin on buttons.
What do you think ?